src/main/java/http/request/RequestParser.java

```diff
@@ -6,68 +6,10 @@
 import java.util.*;
 
 public class RequestParser {
```

Comment:
Does the file have to stay if it has most of the content removed?

Reply:
I'm not sure if i'll use it or not I think it fits in just not sure where yet.